README.md

```diff
@@ -20,25 +20,19 @@
 ###  <div align="left">Turns Data and AI algorithms into full web applications in no time.
 ###  How? Taipy GUI with Taipy Core pops out as a 360° platform to build production-ready web applications</div>
 
-
-
-<br>
 <br>
 
-###  <div align="left">*Open Source, 100% Python*</div>
+###  <div align="left">Pure Python & Open-Source. <div align="left"> How? Taipy pops out as a 360° platform to build production-ready web applications.
 
-
-<br>
-<br>
 <br>
 
 #  <div align="center"> 📊 We make both ends meet ⚙️ </div>
 <br>
  <div align="center">
 
-| TAIPY GUI - the frond-end  | TAIPY Core - the back-end |
+| TAIPY - the frond-end  | TAIPY - the back-end |
 | --------  | -------- |
-|<img src="readme_img/readme_gui_intro.gif" alt="Taipy GUI Animation"  width="100%"/> | <img src="readme_img/readme_core_intro.gif" alt="Taipy Core Animation"  width="100%"/>
+|<img src="readme_img/readme_gui_intro.gif" alt="Taipy FE Animation"  width="100%"/> | <img src="readme_img/readme_be_intro.gif" alt="Taipy BE Animation"  width="100%"/>
```

Comment:
What about having the same illustration as the doc https://docs.taipy.io/en/latest/?

Reply:
to change both the GUI and B-E one? Why not